homeassistant/components/binary_sensor/insteon_plm.py

```diff
@@ -56,32 +61,58 @@ def should_poll(self):
     @property
     def address(self):
         """Return the address of the node."""
-        return self._address
+        return self._device.address.human
 
     @property
-    def name(self):
+    def id(self):
         """Return the name of the node."""
-        return self._name
+        return self._device.id
 
     @property
-    def is_on(self):
-        """Return the boolean response if the node is on."""
-        sensorstate = self._plm.get_device_attr(self._address, 'sensorstate')
-        _LOGGER.info("Sensor state for %s is %s", self._address, sensorstate)
-        return bool(sensorstate)
+    def name(self):
+        """Return the name of the node. (used for Entity_ID)"""
+        if self._newnames:
+            return self._device.id + '_' + self._state.name
+        else:
```

Comment:
No need for `else` after a return.

Reply:
Fixed.  Just FYI, the `newnames` configuration option defaults to `False`. In the future we expect this to default to `True`. The new names provide more description to make automations easier but we did not want to introduce this breaking change in this release. As you correctly stated, there are too many changes already in this release.